src/render.rs

```diff
@@ -1,7 +1,13 @@
-use crate::{resources::AppInstance, scripting::Script, Error, Result};
+use crate::{metadata, resources::AppInstance, scripting::Script, Error, Result};
+use home::home_dir;
+use std::env;
+use tempfile::NamedTempFile;
+
+/// GitHub Registry which contains the `kubecfg` image.
+pub const KUBECFG_REGISTRY: &str = "ghcr.io/kubecfg/kubecfg/kubecfg";
```

Comment:
I initially had this in `lib.rs`, but I'm not sure whether that is the right thing to do.

I've moved it here since it makes sense for having the `kubecfg` image within the `render` portion, i.e. `kubecfg show`, but that can easily change.

Open to suggestions.